### DISCUSSION TITLE
How to create an annotated tag instead lightweight?

### DISCUSSION THREAD

**Discussion by matteovivona** [semantic-release/semantic-release#2200] [Configuration]
I can't find any reference in the documentation to configure .releaserc to create annotated tags instead of lightweight.

Any suggestions?

**Comment by kennethjor** [comment]
It looks like this was originally implemented in https://github.com/semantic-release/semantic-release/pull/1203, but soon reverted in https://github.com/semantic-release/semantic-release/commit/0629f3cd8af8068a0f6daedc1312c37d1936d87f.

I'd also like to create annotated tags instead of lightweight ones. Is there any background on this?

**Comment by ThommyH** [comment]
Seems there is some work here:

https://github.com/semantic-release/semantic-release/pull/1871
